core/Pos/Arbitrary/Core.hs

```diff
@@ -443,13 +442,14 @@ instance Arbitrary G.GenesisCoreData where
                          <$> arbitrary
                          <*> pure (wordILen - a)
                          <*> arbitrary
-                , pure $ G.ExponentialStakes wordILen
+                , pure $ G.ExponentialStakes wordILen (Types.mkCoin 1024)
```

Comment:
1. Which restrictions? What will go wrong if the value is 3?
2. Why not `choose (1000, 2000)` then?